### DISCUSSION TITLE
Arad group at tel aviv

### DISCUSSION THREAD

**Discussion by blorby** [maorcc/citymind_water_meter#25] [Q&A] [answered]
Hi - anyone managed to register or activate the service in Tel Aviv?

**Comment by idodov** [comment, marked as answer]
Starting from November 2022, the new citymind service is open to Tel Aviv (Mei Avivim).

To register you need your customer ID, which is not shown up on the bill, or Mei Avivim personal information (that available on Mei Avivim website).

For this information you need to contact Mei Avivim customer support and ask them for the info. It can be a little annoying because each resident has a unique id, and only one has access to the service.

After you have the customer ID and the property number (that appears on the bill) you can register on Citymind PRO app (not citymind website).

Hope it will be flowless as possible for you.
